accounts/service_test.go

```diff
@@ -444,18 +448,22 @@ func TestAccountService(t *testing.T) {
 			acct := &OffChainBalanceAccount{
 				ID:             testID,
 				Type:           TypeInitialBalance,
-				CurrentBalance: 1234,
+				CurrentBalance: 5000,
 				Invoices: AccountInvoices{
 					testHash: {},
 				},
 				Payments: AccountPayments{
 					testHash: {
 						Status:     lnrpc.Payment_IN_FLIGHT,
-						FullAmount: 1234,
+						FullAmount: 2000,
 					},
 					testHash2: {
 						Status:     lnrpc.Payment_UNKNOWN,
-						FullAmount: 3456,
+						FullAmount: 1000,
+					},
+					testHash3: {
+						Status:     lnrpc.Payment_UNKNOWN,
```

Comment:
Could we also add a case with Status 4 (`lnrpc.Payment_INITIATED`), which is not yet in the proto, but will be for 0.18? Also test the case down below when we receive such an update. I think we may need to invert the inflight check in `paymentUpdate` to check that the status is not success/failure to be able to exit early on the new status code, otherwise status 4 would be treated as a success and may wrongly reduce the balance?

Reply:
Ah nice suggestion! I've updated this with the latest push :)!